### PR TITLE
After installing kernel-ml in CentOS, update grub and reboot

### PR DESCRIPTION
**Pull request by Rumbles**
Fixes scylladb#12

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.idea
+.code
```
Comment (dyasny) [inline .gitignore:2]: No need for these

File: ansible-scylla-node/tasks/RedHat.yml [hunk 2]
```diff
@@ -30,7 +30,37 @@
...
+        backup: yes
+        create: yes
+        regexp: '{{ item.regexp }}'
+        line: '{{ item.line }}'
+        state: present
+      with_items:
+        - regexp: 'GRUB_DEFAULT='
+          line: 'GRUB_DEFAULT=0'
+        - regexp: 'GRUB_TIMEOUT='
+          line: 'GRUB_TIMEOUT=30'
```
Comment (dyasny) [inline ansible-scylla-node/tasks/RedHat.yml:49]: Not sure such a long timeout is required. @vladzcloudius @tarzanek wdyt?

Comment (vladzcloudius) [reply ansible-scylla-node/tasks/RedHat.yml:49]: I agree. But why touching the timeout config in the first place?

**Comment by dyasny** [comment]
@Rumbles Missing another step here - after the step installing the kernel, need to add a register: kernel_installed and only apply the grub changes and reboot when kernel_installed.changed. Otherwise the role will be rebooting nodes every time needlessly.

**Comment by dyasny** [comment]
@Rumbles also, I don't see a reason to touch the grub timeout and make it so long. The sooner the nodes are back from reboot the better

File: .gitignore (hunk 1 above)
Comment (Rumbles) [reply .gitignore:2]: I and other IDE users would disagree with you there

File: ansible-scylla-node/tasks/RedHat.yml [hunk 3]
```diff
@@ -32,7 +32,38 @@
         name: kernel-ml
         state: present
         lock_timeout: 60
-  when: elrepo_kernel == True and install_type == 'online' and (ansible_distribution == "CentOS" or ansible_distribution == "RedHat")
+      register: kernel_installed
+
+    - name: Configure grub2 to boot first entry
```
Comment (tarzanek) [inline ansible-scylla-node/tasks/RedHat.yml:37]: this is needed only for Centos 7, for Centos 8 it's not - will it cause a problem in Centos 8 ?

**Comment by tarzanek** [comment]
merging, I will retest Centos 8

**Pull request merged**